common/lib/permissions.js

```diff
@@ -43,6 +43,21 @@ const contractDeliveryManagerPermissions = [
   'youth_risk_assessment:print',
 ]
 
+const readOnlyPermissions = [
+  'dashboard:view',
+  'allocations:view',
+  'locations:contract_delivery_manager',
```

Comment:
I can't remember exactly what this allows but I'm assuming the users with this role need to be able to see locations in this way? Does it allow them to see *all* locations?

Reply:
`locations:contract_delivery_manager` is in a conditional that leads to getting all locations for all suppliers, `locations:all` gives permission to access the `/all` route

Reply:
And I'm guessing this is something required for this user type?

Reply:
I believe so, from the ticket: This should be a replica of the current PECS CDM role, with access to Incoming and Outgoing moves for all PECS locations, but without the ability to see the maps.